modules/rostests/apitests/ntdll/RtlUnicodeToOemN.c

```diff
@@ -110,7 +110,7 @@ static const struct
 
 START_TEST(RtlUnicodeToOemN)
```

Comment:
Unrelated changes?

Reply:
They fix warnings. You got a problem with that?